### DISCUSSION TITLE
We're sorry, SumatraPDF crashed. × Press 'Cancel' to see crash report.

### DISCUSSION THREAD

**Discussion by cfswb** [community/community#32064] [Other Feature Feedback, Questions, & Ideas]
We're sorry, SumatraPDF crashed.
×
Press 'Cancel' to see crash report.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/sumatrapdfreader/sumatrapdf? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:
